BraveWallet/Crypto/Accounts/AccountsHeaderView.swift

```diff
@@ -8,6 +8,7 @@ import struct Shared.Strings
 
 struct AccountsHeaderView: View {
   @ObservedObject var keyringStore: KeyringStore
+  var cryptoStore: CryptoStore
```

Comment:
You can just pass `SettingsStore` in here, its going to be evaluated immediately anyways since `NavigationLink` isn't lazy